sjb/config/common/test_cases/origin_installed_release.yml

```diff
@@ -3,7 +3,7 @@ parent: 'common/test_cases/origin_release_with_ecosystem.yml'
 overrides:
   provision:
     os: "rhel"
-    stage: "build"
+    stage: "base"
```

Comment:
@adambkaplan i thought the whole point of us changing these jobs was we were moving away from using a common job that no one else depended on, how are we still in this situation?

Reply:
this was per @stevekuznetsov - this parameter dictates which AMI is provisioned to start the test.
are you saying we should avoid the override (which I think is possible).

Reply:
i think i'm asking, "is anyone else using origin_installed_release"?  If so, why aren't they hitting this.  And if not, why are we using it?

Reply:
@bparees to answer your questions:
1. No, no other tests outside our DevEx extended tests use this
2. Looks like other tests use origin_built_installed_release.yaml as the base - testing now to see if this works (scripts are almost identical).

/hold